spec/features/user_post_msg_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'pry'
+feature 'post a message' do
+  scenario 'to see the message with timestamp' do
+    visit '/'
+    fill_in :msg, with: '1st test message'
+    click_on 'send'
+    time = Time.now.strftime("%v %R")
```

Comment:
Don't get why you had the time stamp in here. Might be my misunderstanding, I ll search about it